### PR TITLE
Added metadata to configure remove-live

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR introduces a boolean flag which will be used by sulu to configure the behavior for removing instances of this document.

See https://github.com/sulu/sulu/pull/3302

### CODE REVIEW

File: lib/Metadata.php [hunk 1]
```diff
@@ -44,6 +44,11 @@ class Metadata
     private $fieldMappings = [];
 
     /**
+     * @var bool
+     */
+    private $removeLive = true;
```
Comment (danrot) [inline lib/Metadata.php]: I am somehow missing the connection that this is done automatically when the draft is deleted. Maybe `syncRemoveAcrossWorkspaces`? Although this is quite a long one... Or `syncRemoveLive`?

Comment (danrot) [reply lib/Metadata.php]: Not completely sure about this now, but `removeLive` could also be interpreted as never having a live node at all.

**Review by danrot — CHANGES_REQUESTED** [review]

File: lib/Metadata.php (hunk 1 above)
Comment (wachterjohannes) [reply lib/Metadata.php]: @danrot in my opinion, the `syncRemovLive` sounds good but not perfect ... but i cannot think of any other name. what should i do? (:

Comment (danrot) [reply lib/Metadata.php]: Let's take `syncRemoveLive`, since we don't know a better name...

Comment (wachterjohannes) [reply lib/Metadata.php]: @danrot fixed

**Comment by wachterjohannes** [comment]
@danrot fixed

**Review by danrot — APPROVED** [review]

**Pull request merged**